src/create-model.jl

```diff
@@ -902,8 +926,12 @@ function create_model(
     @timeit to "multi-year investment" begin
         @expression(
             model,
-            accumulate_capacity_simple[y ∈ Y, a ∈ Ai[y]],
-            sum(assets_investment[yi, a] for yi in Yi[a] if starting_year[(y, a)] ≤ yi ≤ y)
+            accumulate_capacity_simple_method[y ∈ Y, a ∈ investable_assets_using_simple_method[y]],
+            graph[a].initial_units[y] +
+            sum(assets_investment[yi, a] for yi in Yi[a] if starting_year[(y, a)] ≤ yi ≤ y) - sum(
+                assets_decommission_simple_method[yi, a] for
+                yi in Yi[a] if starting_year[(y, a)] ≤ yi ≤ y
```

Comment:
- Same here, the year set and asset set should probably be reconsidered.

Reply:
Latest changes account for the reviewed formulation